### PR TITLE
Remove VM confirmation - update title and warning message

### PR DESCRIPTION
**Pull request by thaorell**
Fixes: #1043 

Added a prop for destructive behaviors

screenshot-localhost-3000-2019 07 09-15-56-01

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by bond95** [comment]
ci add to whitelist

File: src/intl/messages.js [hunk 1]
```diff
@@ -417,8 +417,8 @@ export const messages: { [messageId: string]: MessageType } = {
   remoteViewerConnection: 'Remote Viewer Connection',
   remoteViewerAvailable: 'Remote Viewer is available for most operating systems. To install it, search for it in GNOME Software or run the following:',
   remove: 'Remove',
-  removeVm: 'Remove the VM',
-  removeVmQustion: 'Remove the VM?',
+  removeVm: 'Remove VM',
+  removeVmQustion: 'Are you sure you want to remove the VM? All of the data attached to the VM will be removed as well',
```
Comment (sjd78) [inline src/intl/messages.js]:
```suggestion
  removeVmQustion: 'Are you sure you want to remove the VM? All of the data attached to the VM will be removed as well.',
```

**Comment by isaranova** [comment]
ci build please

**Comment by isaranova** [comment]
LGTM

**Comment by sgratch** [comment]
Since this is a duplication of #1077 then I'm closing this PR

**Comment by sjd78** [comment]
@sgratch - Actually there are dialog title and message changes in this PR that are not in #1076.  We should retain the wording changes.

@thaorell - Can you refactor this PR to just include the title change in `VmActions/index.js` and the changes in `src/intl/messages.js`?

**Comment by sgratch** [comment]
> @sgratch - Actually there are dialog title and message changes in this PR that are not in #1076. 

You are right, I will reopen issue #1043 so that we'll remember to handle that

**Comment by bond95** [comment]
@thaorell please, rebase

**Comment by thaorell** [comment]
screenshot-localhost_3000-2019 09 05-14_41_37

Done refactoring

**Comment by isaranova** [comment]
ci build please

**Comment by isaranova** [comment]
LGTM

**Pull request merged**